src/Ticket.php

```diff
@@ -4146,25 +4146,25 @@ public static function getDefaultValues($entity = 0)
 
         // Set default values...
         return  ['_users_id_requester'       => $users_id_requester,
-            '_users_id_requester_notif' => ['use_notification'  => [$default_use_notif],
+            '_users_id_requester_notif' => ['use_notification'  => $default_use_notif,
```

Comment:
This format was correct (even if it does not seesm relevant) and is supported elsewhere in the code. I think it is preferable to fix the location where it is not supported rather than changing this. Indeed, it may have unexpected side effects.

See the `TicketTest::addActorsProvider()` for all expected formats.